### PR TITLE
MapShed BMPs: Update Crop Tillage Practices text

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Updates text descriptions for the three Crop Tillage Practice BMPs.

Connects #2956 

### Demo

screen shot 2018-09-05 at 11 35 19 am

screen shot 2018-09-05 at 11 35 22 am

screen shot 2018-09-05 at 11 35 26 am

## Testing Instructions

* Check out this branch and `bundle`
* Go to a MapShed project and open the Conservation Practices dropdown. Hover over "No Till Agriculture", "Conservation Tillage", and "Reduced Tillage" and observe the new descriptions. Ensure there are no typos.
* Check in various browsers.

### CODE REVIEW

File: src/mmw/js/src/core/modificationConfig.json [hunk 1]
```diff
@@ -141,18 +141,18 @@
         "copyStyle": "deciduous_forest"
     },
     "crop_tillage_no": {
-        "name": "No Till Ag",
-        "summary": "Growing crops or pasture from year to year without tilling or otherwise disturbing the soil, thereby increasing water infiltration, organic matter retention, and nutrient cycling in the soil.",
+        "name": "No Till Agriculture",
+        "summary": "The purpose of this BMP is to leave some residue from harvested crops on the soil surface to reduce soil erosion. This practice is similar to the “Conservation Tillage” and “Reduced Tillage” practices, and only differs with respect to the amount of residue left on the ground. In this case, it is assumed that ground coverage with residual matter is at or greater than 60%.",
```
Comment (kellyi) [inline src/mmw/js/src/core/modificationConfig.json]: Is it possible to break this text across several lines? It'd make commenting on the diff a little easier. 

I did want to make sure that for this and others the proper quote mark to use in the actual description text is `“` and not `"`.

**Comment by kellyi** [comment]
Looks good in Chrome. Testing elsewhere now.

**Comment by kellyi** [comment]
Looks good in Firefox; however I did notice that on a 13" Macbook Pro screen the longer BMP descriptions get cut off at the bottom of the screen and the element's not scrollable:

screen shot 2018-09-05 at 1 42 13 pm

Probably worth making a separate issue for that.

**Comment by kellyi** [comment]
Looks good in both Safari and IE11!

File: src/mmw/js/src/core/modificationConfig.json (hunk 1 above)
Comment (rajadain) [reply src/mmw/js/src/core/modificationConfig.json]: You can't merge strings (or do any other operations) in a JSON file, and adding line breaks within the string would just add extraneous spacing in the resulting text. I'm going to leave it as is.

Regarding the typographer's quotes, I was curious about them myself, but as they are working in all browsers I think we can leave them in.

**Comment by rajadain** [comment]
> the longer BMP descriptions get cut off at the bottom of the screen

That's a good catch, but as it is only the border that is cut off, and most of the content is readable, I'm willing to let that go for now. We may return to it in the future.

**Comment by rajadain** [comment]
Thanks for testing, going to merge this now.

**Pull request merged**